modules/FindFreetype.cmake

```diff
@@ -145,10 +172,9 @@ if(FREETYPE_INCLUDE_DIR_freetype2 AND FREETYPE_H)
   endforeach()
 endif()
 
-
 # handle the QUIETLY and REQUIRED arguments and set FREETYPE_FOUND to TRUE if
 # all listed variables are TRUE
-include(${CMAKE_CURRENT_LIST_DIR}/FindPackageHandleStandardArgs.cmake)
+include(${CMAKE_ROOT}/Modules/FindPackageHandleStandardArgs.cmake)
```

Comment:
I would do just `include(FindPackageHandleStandardArgs)`, similarly for `SelectLibraryConfigurations` above. AFAIK this absolute inclusion is done by CMake developers to prevent accidental module overrides from user-provided module dirs.